pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -768,10 +760,10 @@ spec:
 		"Normal Started",
 		"Normal Running Tasks Completed: 0",
 	}
-	reconciledRun, clients := prt.reconcileRun("foo", "test-pipeline-run-success", wantEvents, false)
+	reconciledRun, clients := prt.reconcileRun(namespace, prName, wantEvents, false)
 
 	// Check that the expected TaskRun was created
-	actual := getTaskRunCreations(t, clients.Pipeline.Actions(), 2)[0]
+	actual := getTaskRunsForPipelineRun(prt.TestAssets.Ctx, t, clients, namespace, prName, 1)[0]
```

Comment:
It looks like it's correct that there's only 1 child TaskRun, but do you know why there were 2 actions here originally? I can see that all the test cases using "simple-hello-world-pipeline" are swapping from 2 actions -> 1 taskrun

Reply:
"2 actions" means there were at least two actions, one is the action of getting the PipelineRun, and the other is the action of creating the TaskRun.